content/python/concepts/casting/casting.md

```diff
@@ -5,23 +5,24 @@ Subjects:
   - 'Computer Science'
   - 'Data Science'
 Tags:
-  - 'Classes'
-  - 'Functions'
+  - 'Data Types'
+  - 'Variables'
+  - 'Variable Types'
 CatalogContent:
   - 'learn-python-3'
   - 'paths/computer-science'
 ---
 
-Casting is a method that converts a variable's data type into another data type. These conversions can be implicit (automatically compiled) or explicit (using built-in functions).
+Casting, also known as type conversion, is a process that converts a variable's data type into another data type. These conversions can be implicit (automatically interpreted) or explicit (using built-in functions).
```

Comment:
I like this new-and-improved first sentence! Could we include this in the meta-`Description` above on line 3?